common.gradle

```
@@ -35,6 +36,11 @@ dependencies {
     implementation AndroidXDependency.CORE_KTX
     implementation AndroidXDependency.APPCOMPAT
     implementation RxJavaDependency.RXJAVA
+    implementation HiltDependency.HILT
+    implementation RetrofitDependency.CONVERTER_MOSHI
+    implementation RetrofitDependency.ADAPTER_RXJAVA3
+
+    kapt HiltDependency.COMPILER
```

```suggestion
    implementation HiltDependency.HILT
    kapt HiltDependency.COMPILER
    
    implementation RetrofitDependency.CONVERTER_MOSHI
    implementation RetrofitDependency.ADAPTER_RXJAVA3
```

관련있는 것끼리 묶는 것은 어떠신가요?

---

https://github.com/kimdoori/GitHubApp/pull/16/commits/10feb5b9eeef4e37826e934330472fed6ef775e5 좋습니다! 반영했습니다!